### PR TITLE
enhance(set rsync api production QOS to high)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

According to issue #2251 
Setting the production QOS to high for rsync jobs.

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/rsync/rsync_api.py [hunk 1]
```diff
@@ -34,9 +34,13 @@ def __init__(self, config: CGConfig):
         self.account: str = config.data_delivery.account
         self.log_dir: Path = Path(config.data_delivery.base_path)
         self.mail_user: str = config.data_delivery.mail_user
-        self.slurm_quality_of_service: str = SLURM_ACCOUNT_TO_QOS[self.account] or SlurmQos.LOW
         self.pipeline: str = Pipeline.RSYNC
 
+    @property
+    def slurm_quality_of_service(self) -> str:
+        """Return the slurm quality of service."""
```
Comment (karlnyr) [inline cg/meta/rsync/rsync_api.py:41]:
```suggestion
        """Return the slurm quality of service depending on the slurm account."""
```

**Review by karlnyr — APPROVED** [review]
neat ^^

File: tests/meta/rsync/test_rsync.py [hunk 2]
```diff
@@ -254,3 +255,23 @@ def test_slurm_rsync_single_case_missing_file(
...
+    rsync_api.account = SlurmAccount.PRODUCTION
+
+    # THEN the quality of service should be set to HIGH
+    assert rsync_api.slurm_quality_of_service == SlurmQos.HIGH
+
+
+def test_slurm_quality_of_service_other(rsync_api: RsyncAPI):
+    # GIVEN an RsyncAPI instance
+
+    # WHEN the account of the api is set to "production
```
Comment (ChrOertlin) [inline tests/meta/rsync/test_rsync.py:273]:
```suggestion
    # WHEN the account of the api is set to development
```

File: tests/meta/rsync/test_rsync.py [hunk 3]
```diff
@@ -254,3 +255,23 @@ def test_slurm_rsync_single_case_missing_file(
...
+    rsync_api.account = SlurmAccount.DEVELOPMENT
+
+    # THEN the quality of service should be set to HIGH
```
Comment (ChrOertlin) [inline tests/meta/rsync/test_rsync.py:276]:
```suggestion
    # THEN the quality of service should be set to LOW
```

File: tests/meta/rsync/test_rsync.py [hunk 4]
```diff
@@ -254,3 +255,23 @@ def test_slurm_rsync_single_case_missing_file(
     # THEN check that an integer was returned as sbatch number
     assert isinstance(sbatch_number, int)
     assert not is_complete_delivery
+
+
+def test_slurm_quality_of_service_production(rsync_api: RsyncAPI):
+    # GIVEN an RsyncAPI instance
+
+    # WHEN the account of the api is set to "production
```
Comment (ChrOertlin) [inline tests/meta/rsync/test_rsync.py:263]:
```suggestion
    # WHEN the account of the api is set to production
```

**Comment by ChrOertlin** [comment]
stage:
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/hiseq.clinical
done.
Log deploy... done.
cg, version 38.2.9

prod:
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/hiseq.clinical
done.
Log deploy... done.
cg, version 38.2.9

**Pull request merged**